packages/webdriverio/src/utils/interception/webdriver.ts

```diff
@@ -16,6 +18,7 @@ export default class WebDriverInterception extends Interception {
     /**
      * allows access to all requests made with given pattern
      */
+    // @ts-ignore
```

Comment:
Couldn't figure out yet how to match with the `calls` type from the class `Interception` (should it match?)

Also this gives the error:
```
'calls' is defined as a property in class 'Interception', but is overridden here in 'WebDriverInterception' as an accessor.ts(2611)
```